rollup.config.js

```diff
@@ -65,8 +65,9 @@ function rewirePreactSubpackages() {
 
           const { dir } = parsePath(filePath);
           const pathToPreact = relativePath(dir, './preact');
+          const escapedPathToPreact = './' + pathToPreact.replace('\\', '\\\\');
```

Comment:
Maybe something like

```
import { sep as pathSeparator } from 'path';

// ...

const pathToPreact = relativePath(dir, './preact').split(pathSeparator).join('/');

return `${ importGroup }./${ pathToPreact }${ afterImport }`;
```

would be a bit more elegant as it would produce something like `./../..` instead of `./..\\..`

Reply:
Stackoverflow agrees: https://stackoverflow.com/questions/53799385/how-can-i-convert-a-windows-path-to-posix-path-using-node-path.

Reply:
There's also a way to do it via the path/posix sub-package I'm pretty sure, but it's only clean if you use the require() way of importing so agreed I'll use that.